tests/ui/single_match.rs

```diff
@@ -397,6 +397,7 @@ pub struct Data([u8; 4]);
 const DATA: Data = Data([1, 2, 3, 4]);
 const CONST_I32: i32 = 1;
 
+// issue #13012
```

Comment:
Really nitty nit, could this be a Github link so that people can click it from their editors instead of manually searching for it?